spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala

```diff
@@ -77,21 +76,26 @@ class Graph[T: ClassTag](val inputs : Seq[ModuleNode[T]],
       val node = forwardExecutions(i)
       val nodeInput = if (node.prevNodes.isEmpty && !node.element.isInstanceOf[WithoutInput]) {
         inputData(node, input)
-      } else if (node.prevNodes.length == 1) {
-        node.prevNodes.head.element.output.toTensor[T]
       } else {
-        seqToTable(node.prevNodes.map(_.element.output))
+        var toOffset = 1
+        val prevTensors = node.prevNodesAndEdges.flatMap(n => {
```

Comment:
It seems Graph outputs one big flat table. Say a Graph has multiple output nodes, and one of the output node outputs unknown number of tensors, then the user of this Graph seems have no way to determine which tensors in the output table belongs to which output node.

Reply:
fixed